### PR TITLE
WT-10764 Adding a message to indicate that rollback to stable has fin…

### PR DESCRIPTION
**Pull request by luke-pearson**
…ished

### CODE REVIEW

File: src/rollback_to_stable/rts_api.c [hunk 1]
```diff
@@ -195,6 +195,9 @@ __rollback_to_stable(WT_SESSION_IMPL *session, const char *cfg[], bool no_ckpt)
     WT_STAT_CONN_SET(session, txn_rollback_to_stable_running, 1);
     WT_WITH_CHECKPOINT_LOCK(
       session, WT_WITH_SCHEMA_LOCK(session, ret = __rollback_to_stable_int(session, no_ckpt)));
+
+    __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
```
Comment (agorrod) [inline src/rollback_to_stable/rts_api.c:199]: Would it make sense to add a similar message at around line 183 indicating that RTS is starting?

**Review by agorrod — COMMENTED** [review]
One question, and I've asked @korteland to be a reviewer as well, since some of the RTS messaging is linked into a new tool, and I'd like his input as to whether there is anything further that needs to be done.

File: src/rollback_to_stable/rts_api.c (hunk 1 above)
Comment (luke-pearson) [reply src/rollback_to_stable/rts_api.c:199]: On line 77 of rts_api.c there is a message which says: "performing rollback to stable with stable_timestamp=" which I think should suffice. @kommiharibabu would you agree?

Comment (luke-pearson) [reply src/rollback_to_stable/rts_api.c:199]: It looks something like this, which is from one of the unit tests that failed on the ticket:
```
[2023/03/17 03:21:03.856] [1679023257:917968][33448:0x7f699a135800], test_rollback_to_stable25.test_rollback_to_stable25.test_rollback_to_stable25(10l.upd.20u.nil.30l.del.chk20.roll15), txn rollback_to_stable: [WT_VERB_RTS][DEBUG_1]: [INIT] performing rollback to stable with stable_timestamp=(0, 15) and oldest_timestamp=(0, 2)
[2023/03/17 03:21:03.856] [1679023257:918009][33448:0x7f699a135800], test_rollback_to_stable25.test_rollback_to_stable25.test_rollback_to_stable25(10l.upd.20u.nil.30l.del.chk20.roll15), txn rollback_to_stable: [WT_VERB_RTS][DEBUG_1]: [TREE_SKIP] file:rollback_to_stable25.wt: tree skipped with durable_timestamp=(0, 10) and stable_timestamp=(0, 15) or txnid=6
[2023/03/17 03:21:03.856] [1679023257:918024][33448:0x7f699a135800], test_rollback_to_stable25.test_rollback_to_stable25.test_rollback_to_stable25(10l.upd.20u.nil.30l.del.chk20.roll15), txn rollback_to_stable: [WT_VERB_RTS][DEBUG_1]: [END] rollback to stable finished
```

Should I add a comment explaining why we don't have a start message?

**Review by korteland — COMMENTED** [review]
Can you please add the new log line to the RTS verifier? Should be easy to add a no-op version (there's going to be a separate ticket for using these lines to do some useful verifying):

```
diff --git a/tools/rts_verifier/checker.py b/tools/rts_verifier/checker.py
index c04e5292f..bedd9e8ee 100644
--- a/tools/rts_verifier/checker.py
+++ b/tools/rts_verifier/checker.py
@@ -215,3 +215,7 @@ class Checker:
     def __apply_check_hs_truncated(self, operation):
         # TODO expand this out
         pass
+
+    def __apply_end(self, operation):
+        # TODO expand this out
+        pass
diff --git a/tools/rts_verifier/operation.py b/tools/rts_verifier/operation.py
index d77849231..2ca4e46af 100644
--- a/tools/rts_verifier/operation.py
+++ b/tools/rts_verifier/operation.py
@@ -36,6 +36,7 @@ class OpType(Enum):
     FILE_SKIP = 27
     SKIP_DAMAGE = 28
     HS_TRUNCATED = 29
+    END = 30

 class Operation:
     def __init__(self, line):
@@ -464,3 +465,6 @@ class Operation:

         matches = re.search('btree=(\d+)', line)
         self.btree_id = int(matches.group(1))
+
+    def __init_end(self, line):
+        self.type = OpType.END
```

**Comment by luke-pearson** [comment]
> Can you please add the new log line to the RTS verifier? Should be easy to add a no-op version (there's going to be a separate ticket for using these lines to do some useful verifying):

[7c2fc04](https://github.com/wiredtiger/wiredtiger/pull/8934/commits/7c2fc04ddc2c9b730d99197bbf99813ce7f071f1)

**Review by korteland — APPROVED** [review]

File: src/rollback_to_stable/rts_api.c (hunk 1 above)
Comment (sueloverso) [reply src/rollback_to_stable/rts_api.c:199]: Why not just add the word "Start" so that it reads "Start performing rollback..." and it is very clear.

Comment (kommiharibabu) [reply src/rollback_to_stable/rts_api.c:199]: Yes, the existing verbose message with more details is good as the start message. I feel there is no need to add a new start message.

Comment (luke-pearson) [reply src/rollback_to_stable/rts_api.c:199]: I'll add the word "Start" as Sue suggested.

**Review by sueloverso — APPROVED** [review]

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Review by agorrod — APPROVED** [review]
lgtm - thanks @luke-pearson

**Review by korteland — APPROVED** [review]

**Pull request merged**